### DISCUSSION TITLE
spawning 2 monitors "fullscreen"

### DISCUSSION THREAD

**Discussion by zeph** [rxhanson/Rectangle#1400] [General]
I have 2x 32:9 (ratio) monitors, one above the other... 
I'd love to be able to have a function "stretch over the full Space"

the option "Displays have separate Spaces" is obvious `off`, I can stretch
 the window across both manually, having literally a huge 5k monitor

would be nice not to have to do it manually... but with a keyboard shortcut
(as double clicking, or using the "green" button would use just one screen)

**Comment by rxhanson** [comment]
You might be interested in these actions, which were a contribution to a recent release: https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#add-doublinghalving-window-sizing-commands

The doubling actions will go beyond the display boundary when you have `Displays have separate Spaces` off.

**Comment by asarturas** [comment]
@rxhanson this works with latest rectangle (Version 0.79 (85)), but dues not seem to work with latest Rectangle Pro (Version 3.0.26 (167)). Do you have a roadmap of when is it coming to pro?

update: found that it works, I needed to replace `com.knollsoft.Rectangle` with `com.knollsoft.Hookshot` as described in https://github.com/rxhanson/RectanglePro-Community?tab=readme-ov-file#terminal-commands
